airflow/dag_processing/bundles/manager.py

```diff
@@ -54,44 +100,20 @@ def parse_config(self) -> None:
         if self._bundle_config:
             return
 
-        backends = conf.getjson("dag_processor", "dag_bundle_config_list")
-
-        if not backends:
+        config_list = conf.getjson("dag_processor", "dag_bundle_config_list")
+        if not config_list:
             return
-
-        if not isinstance(backends, list):
-            raise AirflowConfigException(
-                "Bundle config is not a list. Check config value"
-                " for section `dag_processor` and key `dag_bundle_config_list`."
-            )
-
-        if any(b["name"] == "example_dags" for b in backends):
+        if not isinstance(config_list, list):
```

Comment:
Should we move this into the validate function too?

Reply:
I assumed mypy might get grumpy when we iterate below so i left it outside

Reply:
🤷 it's probably fine, but doesn't matter either way.

Reply:
turns out, it ain't fine :) 

<img width="657" alt="image" src="https://github.com/user-attachments/assets/c5bf19d5-4e1d-4465-a726-e0e8694cd897" />